siliconcompiler/tools/magic/magic.py

```diff
@@ -20,14 +20,21 @@ def make_docs():
 
     chip = siliconcompiler.Chip('<design>')
     chip.load_pdk('skywater130')
-    chip.set('arg','index','<index>')
+    index = '<index>'
+    flow = '<flow>'
+    chip.set('arg','index',index)
+    chip.set('option', 'flow', flow)
 
     # check drc
-    chip.set('arg','step','drc')
-    setup(chip)
+    from tools.magic.drc import setup as setup_drc
+    chip.set('arg','step', 'drc')
+    chip.set('flowgraph', flow, 'drc', index, 'task', 'drc')
+    setup_drc(chip)
```

Comment:
why is this needed?

Reply:
Good question. The existing `make_doc` function called both DRC and LVS setup methods, possibly to populate all of the associated parameters for the Sphinx auto-generation?

Reply:
Lets make the sphinx update a separate PR